docs/notes/todo.md

```diff
@@ -0,0 +1,45 @@
+# Todo
+
+
+## Critical Path Tasks
+
+- [ ] Test and debug Registry routines
+- [ ] Implement file cache
+- [ ] Implement paging
+
+
+## These Tasks Would Be Nice
+
+- [ ] Separate HAL into KM DLL
+
+
+## Function Groups Totally or Partially Unimplemented
+
+- [ ] Dma functions (see hal/x86/dma.c)
+- [ ] Shutdown support (see ex/power.c)
```

Comment:
The Executive shutdown routines in `shutdown.c` (ex `power.c`) depend exclusively on Power Manager to inquire a shutdown request, There's nothing more that you can do within that file, so please remove this.

Reply:
Todo now gives a link to this wiki article: https://reactos.org/wiki/Missing_ReactOS_Functionality